### ISSUE TITLE
Translation / Add support for custom translations in @ory/elements-markup

### ISSUE THREAD

**Issue by veteran29** [ory/elements#166]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/elements/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/elements/blob/master/CONTRIBUTING.md).
- [ ] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe your problem

We're deploying self-hosted Ory Kratos, to speed up the project we've used the node self-serve UI from Ory.
During the implementation we've found out that unlike to its React counterpart you can't pass `customTranslations` to `@ory/elements-markup` components.

### Describe your ideal solution

Addition of `customTranslations` parameter to elements-markup [component-wrapper](https://github.com/ory/elements/blob/main/src/markup-components/component-wrapper.tsx#L19) intl provider.

### Workarounds or alternatives

I don't see other viable alternative.

### Version

0.1.0-beta.11

**Comment by Benehiko** [comment]
Yes, you're right. We've only added the custom language support recently. 

PRs are welcome!

**Comment by dmeijboom** [comment]
Would be great if the `stale` label can be removed. We're comitted to adding support for custom translations in `@ory/elements-markup`.
